src/script/cpp_api/s_async.cpp

```diff
@@ -35,6 +35,16 @@ extern "C" {
 #include "common/c_packer.h"
 #include "lua_api/l_base.h"
 
+LuaJobInfo::LuaJobInfo(std::string &&func, std::string &&params, const std::string &mod_origin):
+	function(func), params(params), mod_origin(mod_origin)
+{}
+
+LuaJobInfo::LuaJobInfo(std::string &&func, PackedValue *params, const std::string &mod_origin):
+	function(func), mod_origin(mod_origin)
+{
+	params_ext.reset(params);
+}
```

Comment:
note: could be inlined

Reply:
I move the definitions to the header file. I assume I don't have to explicitly declare these as `inline`?

Reply:
Indeed.